### ISSUE TITLE
Using pyenv with package errors on tkinter on Linux to use MouseInfo.

### ISSUE THREAD

**Issue by progsmile** [asweigart/pyautogui#629]
Hello dear developers! I'm using python with `pyenv`. And when I start the script in shows me 
```
NOTE: You must install tkinter on Linux to use MouseInfo. Run the following: sudo apt-get install python3-tk python3-dev
```
However I've already installed these packages.
How can I go on? I need your assistance with this stuff and I am able to provide any information you need to go development.
Thanks!

--
Python 3.9.7
PyAutoGUI==0.9.52
Distributor ID:	Debian
Description: Debian GNU/Linux 9.13 (stretch)
Release:	9.13

**Comment by adriangabura** [comment]
Yeah, good luck with that I say. I don't want to say much but this isn't maintained properly.

**Comment by progsmile** [comment]
No updates from my side
